src/lib/shareTask.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    IHttp,
+    IMessageBuilder,
+    IModify,
+    IModifyCreator,
+    IPersistence,
+    IRead,
+} from "@rocket.chat/apps-engine/definition/accessors";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+import { IUIKitResponse, TextObjectType, UIKitViewSubmitInteractionContext , ButtonStyle, UIKitBlockInteractionContext} from '@rocket.chat/apps-engine/definition/uikit';
+import {
+    ISlashCommand,
+    SlashCommandContext,
+} from "@rocket.chat/apps-engine/definition/slashcommands";
+import { IUIKitBaseIncomingInteraction, IUIKitViewSubmitIncomingInteraction } from "@rocket.chat/apps-engine/definition/uikit/UIKitIncomingInteractionTypes";
+import { ICreateTaskState } from "../facade/IClickUpService";
+import { IUser } from "@rocket.chat/apps-engine/definition/users";
+import { getAccessTokenForUser } from "../storage/users";
+import { ModalsEnum } from "../enums/Modals";
+import { MiscEnum } from "../enums/Misc";
+
+export async function shareTask({
+    context,
+    data,
+    room,
+    read,
+    persistence,
+    modify,
+    http,
+}: {
+    context: UIKitBlockInteractionContext,
+    data: IUIKitBaseIncomingInteraction,
+    room: IRoom;
+    read: IRead;
+    persistence: IPersistence;
+    modify: IModify;
+    http: IHttp;
+}) {
+    const user: IUser = context.getInteractionData().user;
+    const token = await getAccessTokenForUser(read, user);
+    const task_id = context.getInteractionData().value;
+    const headers = {
+        Authorization: `${token?.token}`,
+    };
+
+    const response = await http.get(`https://api.clickup.com/api/v2/task/${task_id}/`,{ headers });
+    
+    if(response.statusCode==200) {
```

Comment:
change the 200 code to the enum, you can do it by: 
`import { HttpStatusCode } from '@rocket.chat/apps-engine/definition/accessors';`